tests/test_import/module_recursive_package_here.R

```diff
@@ -0,0 +1,7 @@
+# Import utility function or package using import::here() is the recommended usage
+import::here(knitr, normal_print)
```

Comment:
I wonder, generally, if we should rely on "suggest" packages, other than testthat, for tests?

Reply:
I believe that is what the guidelines (from documentation on testthat, I think) suggest. And the CRAN servers ensure that suggests packages are installed before running the tests.

Given the purpose of the package, it's impossible to perform tests from packages without relying on other packages. But an end user does not really need any specific packages, and moving the dependency to Import would force the end users to install them along with import.

Reply:
Not too big a deal - either way. But one could use packages that are shipped with R, eg utils, tools, methods, ...

Reply:
I see, yes, that's probably true, but regardless, I'd vote to keep it as is for now :-)

Reply:
✅